plugins/modules/vmware_guest_disk.py

```diff
@@ -142,9 +142,15 @@
          default: True
        filename:
          description:
-           - Existing disk image to be used. Filename must already exist on the datastore.
+           - Existing disk image to be used. Filename must already exist on the datastore. Can be overridden by specifying C(force_create).
            - Specify filename string in C([datastore_name] path/to/file.vmdk) format. Added in version 2.10.
          type: str
+       force_create:
+         description:
+           - Force the creation of the file specified in C(filename) if it does not exist.
```

Comment:
I think here the description should be more accurate, since C(filename) will not be checked for existence in the code, when C(force_create) is set to True, then no matter C(filename) exists or not, the new disk file will be created.